lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -164,6 +224,7 @@ private class MutationExtension<T : Any>(val spec: MutationExtensionSpec<T>) : E
             viewHolder.adapterPosition,
             onDataItem = {
               if (spec.canSwipeToRemoveItem(it)) {
```

Comment:
1 nit here - `canSwipeToRemove` should also be renamed `canSwipe`

As a possible followup should we allow a `canSwipeLeft` and `canSwipeRight`?

Reply:
Yes, we should return either the side or both.
I might declare an enum to avoid ints.

Reply:
+1 to enums